op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```
@@ -0,0 +1,217 @@
+
+You can use the automated `op-monitorism/faultproof_withdrawals/runbooks/automated/triage_potential_attack_event.ipynb` runbook for this process.
+
+
+### faultproof-potential-withdrawal-forgery-detected
+
+| **Network** | **Severity Level** | **Impact**                      | **Reaction**                                             | **Actions**                                                                                                                                       |
+|-------------|--------------------|---------------------------------|----------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Mainnet     | SEV3               | No immediate impact                       | Investigate the attack, monitor how the attack is proceeding to make sure to be ready to react quickly in case of success                               |  If new type of attack, write a report and eventually create a feature request for improving monitorism monitoring capabilities                                                                            |
+| Sepolia     | SEV3               | No immediate impact                       | Investigate the attack, monitor how the attack is proceeding to make sure to be ready to react quickly in case of success                              | If new type of attack, write a report and eventually create a feature request for improving monitorism monitoring capabilities |
+
+##### Alert Description
```

Markdownlint

Expected: h4; Actual: h5
Heading levels should only increment by one level at a time

(MD001, heading-increment)